android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java

```diff
@@ -400,6 +400,37 @@ static void trim(ReadableMap options, final Promise promise, ReactApplicationCon
     executeFfmpegCommand(cmd, tempFile.getPath(), ctx, promise, "Trim error", null);
   }
 
+  static void pad(ReadableMap options, final Promise promise, ReactApplicationContext ctx) {
```

Comment:
I can't find the React implementation of this method but if I expect it to be standalone functionalities. But if we plan to invoke this method after/before the trim method, we might want to integrate the methods so it only runs ffmpeg for a single video once and not twice.

Reply:
https://github.com/relivecc/skyhawk-mobileapp/pull/3386

Reply:
👍